### ISSUE TITLE
phpunit scripts are missing

### ISSUE THREAD

**Issue by nymo** [propelorm/Propel2#1039]
Since this pull request (Migrate from circle to travis #1000) the following files are missing:

tests/bin/phpunit.mysql.sh	
tests/bin/phpunit.pgsql.sh	
tests/bin/phpunit.sqlite.sh
tests/bin/phpunit.agnostic.sh
tests/bin/setup.sqlite.sh

Is there are reason why? If yes then the documentation has to be updated :)

**Comment by marcj** [comment]
Oh, no, they should be recovered. my mistake, have overlooked it :( @Big-Shark can you please add them back? :)

**Comment by Big-Shark** [comment]
Maybe simply update the documentation?
Now the tests are run as:
```
./vendor/bin/phpunit -c tests/agnostic.phpunit.xml
./vendor/bin/phpunit -c tests/mysql.phpunit.xml 
./vendor/bin/phpunit -c tests/pgsql.phpunit.xml
./vendor/bin/phpunit  -c tests/sqlite.phpunit.xml
```

**Comment by nymo** [comment]
Starting tests on the cmd line is more comfortable with the .sh files. Sure I can create an alias but that is somehow just a workaround for me. What speaks against the old sh files?

**Comment by Big-Shark** [comment]
@nymo I just don't see the point of one-line .sh files

**Comment by marcj** [comment]
The point is, that you can just tab through all possible commands to execute. the way with phpunit -c filex is you have always to know how phpunit works (with -c) and that you need a config file. also if you need to pass extra data like "DB_PW=mypw DB_USER=marc phpunit" then the command line is shorter. I prefer to have those shell scripts :)

**Comment by motin** [comment]
When I look in the current master, the phpunit scripts seem to be in place: https://github.com/propelorm/Propel2/tree/9aff652f9af8bc424df6564ea69a1489e5f0dc21/tests/bin

This issue should be closed?

**Comment by Big-Shark** [comment]
@motin I merged #1000 and #1040 and create a new PR #1059, I think we need close this issue.

**Comment by nymo** [comment]
@motin It's in master because marcj reverted a commit. But we can close this because #1059 has the missing files and current master also.
